sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -195,7 +195,11 @@ const alwaysBeExistingRule = "alwaysbeexistingrule";
         }
 
         should.equal(error.statusCode, 404, "Error must not be undefined");
```

Comment:
The message here is not valid for the check being made.

```suggestion
        should.equal(error.statusCode, 404, "Unexpected status code found.");
```

Reply:
Same for a few other places in this file.

Reply:
Updated